### ISSUE TITLE
missing types in package

### ISSUE THREAD

**Issue by mustafaskyer** [stackframe-projects/stack#23]
After installed package, i can't get declaration from the package it self, hence it's really hard to use or expect exported types wether it's missing or valid!

this is what i got 
```ts
The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("@stackframe/stack")' call instead.
  To convert this file to an ECMAScript module, change its file extension to '.mts', or add the field `"type": "module"`
```

I tried to change the file extension to `.mts` but still can not get types for e.g StackServerApp

**Comment by mustafaskyer** [comment]
this on the latest version `v/2.3.5`

**Comment by N2D4** [comment]
Thanks for the bug report! Working on a fix 👍

**Comment by mustafaskyer** [comment]
Thanks for your fast reply, appreciated 🙏

**Comment by N2D4** [comment]
Should be fixed in v2.3.7, can you try that?
